kuma/users/views.py

```diff
@@ -175,36 +177,23 @@ def profile_edit(request, username):
 
         # Don't validate if the username hasn't changed so people
         # can keep already existing invalid usernames.
-        posted_username = request.POST.get('profile-username', None)
-        if posted_username is not None:
-            username_changed = request.user.username != posted_username
-        else:
-            username_changed = False
+        # posted_username = request.POST.get('user-username', None)
+        # if posted_username is not None:
+        #     username_changed = request.user.username != posted_username
+        # else:
+        #     username_changed = False
 
-        if profile_form.is_valid() and newsletter_form.is_valid():
-            if username_changed:
-                profile.user.username = profile_form.cleaned_data['username']
-                profile.user.save()
+        if user_form.is_valid() and newsletter_form.is_valid():
+            # if username_changed:
+            #     edit_user.username = user_form.cleaned_data['username']
+            #     edit_user.save()
```

Comment:
Oh crap, I forgot I commented that out, damn. Hmmm, I think I'll go through the prod dump and see how many usernames don't match the username regex now. Maybe we can get rid of it.